### DISCUSSION TITLE
Auto-close dropdown on scroll & allow outside click

### DISCUSSION THREAD

**Discussion by philipaarseth** [radix-ui/primitives#1254] [Help] [answered]
Hi,

I couldn't find anything in the documentation about any props that let you change how the dropdown handles scroll.
Is there any way to either allow scroll while dropdown is open or auto-close the dropdown if a scroll begins?

On a separate note, is there any way to allow click outside to open another dropdown? Currently if you have a dropdown open and would like to open another one, you need two clicks.

**Comment by jjenzz** [comment, marked as answer]
i think you might be looking for the `modal` prop? https://www.radix-ui.com/docs/primitives/components/dropdown-menu#root

**Reply by philipaarseth** [reply]
Yes, that's the one I'm looking for,  thank you! Should it work with sticky content? Mine will follow scroll regardless of where you use it.

**Reply by benoitgrelard** [reply]
You might be referring to this, in which case it's an open issue: https://github.com/radix-ui/primitives/issues/781

**Reply by philipaarseth** [reply]
Right yes, looks to be the same problem I'm facing.

**Reply by benoitgrelard** [reply]
If you have a browse through the issue, there are few workarounds.
